base-dev.py

```diff
@@ -97,7 +97,7 @@
     # Warning: installing slurm-pmi2 before the compilers and jedi-stack means
     # that some options are not selected, such as support for hdf5, pmix, and netloc
     # If we're preformance tuning, we may want to re-install this at later stage
-    Stage0 += slurm_pmi2(version='19.05.4',ospackages=['libgtk-3-0','libgtk-3-dev',
+    Stage0 += slurm_pmi2(version='20.02.7',ospackages=['libgtk-3-0','libgtk-3-dev',
```

Comment:
@mer-a-o @rickgrubin - would you mind reviewing this so we can merge?  It's a trivial review - this is the only line that has really changed.   The Dockerfiles are automatically generated.    Version 19.05.4 is not even available on the download site anymore so this change is needed.